lib/core/device.js

```diff
@@ -161,6 +152,17 @@ _.forEach(_.keys(device.deviceComms), (driverId) => {
 device.deviceInfoCache = {};
 
 device.init = (options, cb) => {
+  try {
+    /* eslint-disable global-require, import/no-unresolved, import/extensions */
+    device.deviceDrivers.Tandem = require('../drivers/tandem/tandemDriver');
+    device.deviceDrivers.TandemJellyfish = require('../drivers/tandem/jellyfish/tandemDriver');
+  } catch (e) {
```

Comment:
Yeah, not really of any use, as it will just say the file can't be found if you don't have access to the private submodule. I'll rename it to `err` for consistency.